### DISCUSSION TITLE
Question about QCQP in JAXOpt

### DISCUSSION THREAD

**Discussion by athindran** [google/jaxopt#477] [Ideas]
Hi, Is it possible to solve convex QCQPs with JAXOpt currently? If not, how difficult would it be to write a QCQP solver which can be integrated into a JAX computational graph?

**Comment by mblondel** [comment]
Hello @athindran 

We don't have immediate plans to add support for QCQPs. Usually, adding new solvers can take a fair amount of work as we want to ensure that they are fully tested and documented.
